zips/zip-0227.rst

```diff
@@ -322,19 +322,31 @@ For the ``IssueBundle``:
 
 **Note:** that the commitment is not included in the ``IssuanceAction`` itself. As explained below, it is computed later by the validators and added to the note commitment tree.
 
-Specification: Global Issuance State
-====================================
+Specification: Reference Notes and Global Issuance State
+========================================================
+
+Reference Notes
+---------------
+
+A reference note for a Custom Asset MUST be included by the issuer as the first Note in the Action of the Issuance Bundle where that Custom Asset is being issued for the first time.
```

Comment:
Consider
"A reference note, $Note^{Issue}_{ref}$.."
Or

Reply:
"A reference note, $Note_{ref}$.."
instead of `.ref_note`

Reply:
I made it `\mathsf{note_{ref}}`, like the second comment. I don't need to add that notation anywhere except for the issuance state, because the reference note is just an issue note, until it gets added to the appropriate tuple in the issuance state.